### PR TITLE
Suriya make catkin package

### PR DESCRIPTION
**Pull request by SuriyaNitt**

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package format="2">
+    <name>dep_dcnv2</name>
+    <version>0.0.2</version>
+    <description>C Numpy package</description>
+    <url type="website">www.github.com/andrasta/DCNv2</url>
+    <maintainer email="eng@cyngn.com">Cyngn</maintainer>
+    <author email="eng@cyngn.com">Cyngn</author>
+    <license>BSD3</license>
```
Comment (mc-android-developer) [inline package.xml]: Why it's BSD3?

File: package.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package format="2">
+    <name>dep_dcnv2</name>
+    <version>0.0.2</version>
+    <description>C Numpy package</description>
```
Comment (JscLee) [inline package.xml]: Looks like the description is for cnpy

File: package.xml (hunk 1 above)
Comment (SuriyaNitt) [reply package.xml]: https://tldrlegal.com/license/bsd-3-clause-license-(revised)

File: package.xml (hunk 2 above)
Comment (SuriyaNitt) [reply package.xml]: ok let me fix

Comment (SuriyaNitt) [reply package.xml]: done.

**Pull request merged**